superbench/benchmarks/micro_benchmarks/cublaslt_function.py

```diff
@@ -57,9 +92,10 @@ def add_parser_arguments(self):
             help='Number of steps to measure.',
         )
         self._parser.add_argument(
-            '--in_type',
+            '--in_types',
             type=str,
-            default='fp8e4m3',
+            nargs='+',
+            default=['fp8e4m3'],
             required=False,
             help='Input data type, supports {}.'.format(' '.join(self._in_types)),
```

Comment:
Shall we revise the help to explain multiple support.

Reply:
yes, updated